src/main/flight/gps_rescue.c

```diff
@@ -294,7 +294,7 @@ void rescueStop()
 void idleTasks()
 {
     // Do not calculate any of the idle task values when we are not flying
-    if (!ARMING_FLAG(ARMED)) {
+    if (!ARMING_FLAG(ARMED) || calculateThrottleStatus() == THROTTLE_LOW) {
```

Comment:
Are you sure that this is safe? `THROTTLE_LOW` can also occur when flying.

Reply:
I do not see why it could be an issue -- if you are not hitting throttle, calculating hover throttle has no value, and also if you are at throttle low you're also not gaining altitude, in any scenario there would be no value to calculating idle tasks when the quad isn't doing powered flight.

Reply:
Ok, good point. The check for `!ARMING_FLAG(ARMED) || calculateThrottleStatus() == THROTTLE_LOW` just looks like it's a weird check. Maybe add a comment explaining that these calculations won't be needed when in-flight with throttle low - this is also to make sure the next developer changing this function understands this and acts accordingly.